### PR TITLE
DNU doesn't like semi-colons anymore.  It uses multiple args for the same flag.

### PR DESCRIPTION
**Pull request by adamhathcock**
…

### CODE REVIEW

**Comment by adamhathcock** [comment]
Oh yeah.  I made this on to main to try to get it as a patched version against 0.10.1

File: src/Cake.Common/Tools/DNU/Build/DNUBuilder.cs [hunk 1]
```diff
@@ -62,16 +62,22 @@ private ProcessArgumentBuilder GetArguments(string path, DNUBuildSettings settin
 
             // List of frameworks
             if (settings.Frameworks != null && settings.Frameworks.Count > 0)
-            {
-                builder.Append("--framework");
-                builder.AppendQuoted(string.Join(";", settings.Frameworks));
+            {
+                foreach (var source in settings.Frameworks)
```
Comment (gep13) [inline src/Cake.Common/Tools/DNU/Build/DNUBuilder.cs]: Should the variable here not be `framework`?

File: src/Cake.Common/Tools/DNU/Pack/DNUPacker.cs [hunk 2]
```diff
@@ -62,16 +62,22 @@ private ProcessArgumentBuilder GetArguments(string path, DNUPackSettings setting
 
             // List of frameworks
             if (settings.Frameworks != null && settings.Frameworks.Count > 0)
-            {
-                builder.Append("--framework");
-                builder.AppendQuoted(string.Join(";", settings.Frameworks));
+            {
+                foreach (var source in settings.Frameworks)
```
Comment (gep13) [inline src/Cake.Common/Tools/DNU/Pack/DNUPacker.cs]: Should the variable here not be `framework`?

File: src/Cake.Common/Tools/DNU/Pack/DNUPacker.cs [hunk 3]
```diff
@@ -62,16 +62,22 @@ private ProcessArgumentBuilder GetArguments(string path, DNUPackSettings setting
...
+                }
             }
 
             // List of configurations
             if (settings.Configurations != null && settings.Configurations.Count > 0)
-            {
-                builder.Append("--configuration");
-                builder.AppendQuoted(string.Join(";", settings.Configurations));
+            {
+                foreach (var source in settings.Configurations)
```
Comment (gep13) [inline src/Cake.Common/Tools/DNU/Pack/DNUPacker.cs]: Should the variable here not be `configuration`?

File: src/Cake.Common/Tools/DNU/Restore/DNURestorer.cs [hunk 4]
```diff
@@ -58,15 +58,21 @@ private ProcessArgumentBuilder GetArguments(FilePath path, DNURestoreSettings se
...
+                    builder.AppendQuoted(source);
+                }
             }
 
             // List of fallback package sources
             if (settings.FallbackSources != null && settings.FallbackSources.Count > 0)
             {
-                builder.Append("--fallbacksource");
-                builder.AppendQuoted(string.Join(";", settings.FallbackSources));
+                foreach (var source in settings.FallbackSources)
```
Comment (gep13) [inline src/Cake.Common/Tools/DNU/Restore/DNURestorer.cs]: Should the variable here not be `fallbacksource`?

File: src/Cake.Common/Tools/DNU/Build/DNUBuilder.cs [hunk 5]
```diff
@@ -62,16 +62,22 @@ private ProcessArgumentBuilder GetArguments(string path, DNUBuildSettings settin
...
+                }
             }
 
             // List of configurations
             if (settings.Configurations != null && settings.Configurations.Count > 0)
-            {
-                builder.Append("--configuration");
-                builder.AppendQuoted(string.Join(";", settings.Configurations));
+            {
+                foreach (var source in settings.Configurations)
```
Comment (gep13) [inline src/Cake.Common/Tools/DNU/Build/DNUBuilder.cs]: Should the variable here not be `configuration`?

**Comment by gep13** [comment]
@adamhathcock Thank you for your submission.  We have an upcoming release (within the next 2 weeks) will would likely be within the same timeframe as a hotfix release onto the main branch.  At this time, I think it would make more sense to target the develop branch for this PR, and we can include it in the 0.11.0 release.

I have left some comments, if you could fix them up, that would be great!  Thanks

**Pull request closed without merging**